install_config/storage_examples/direct_storage.adoc

```diff
@@ -0,0 +1,481 @@
+= Direct access to storage using two methods
```

Comment:
Well, one uses the plugin directly and completely bypasses the pv/pvc, that is why I left it out of the title